### PR TITLE
Fixtures & House/Senate parser

### PR DESCRIPTION
**Pull request by arpariseau**
Pulled Propublica API data using Postman. Built out a parser that turns the JSON from the API into our objects for House members and senators. Also includes minor maintenance related to the database to make things work correctly as well as webmock setup.

Bills still a WIP.

### CODE REVIEW

File: app/services/propublica_parser.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    clean_hash = delete_unneeded_sen_attr(sen_hash)
+    clean_hash[:govtrack_id] = clean_hash[:govtrack_id].to_i
+    clean_hash[:last_updated] = clean_hash[:last_updated].to_datetime
+    clean_hash[:dob] = clean_hash.delete(:date_of_birth)
+    clean_hash[:congress_id] = clean_hash.delete(:id)
+    clean_hash[:missed_votes_percentage] = clean_hash.delete(:missed_votes_pct)
+    clean_hash[:votes_with_percentage] = clean_hash.delete(:votes_with_party_pct)
+    clean_hash[:votes_without_party_percentage] = clean_hash.delete(:votes_against_party_pct)
+    clean_hash
+  end
```
Comment (Rostammahabadi) [inline app/services/propublica_parser.rb:52]: Nice helper methods to set up the clean hash

File: app/services/propublica_parser.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+  def delete_unneeded_rep_attr(rep_hash)
+    unneeded_attr = [:title, :short_title, :api_uri, :middle_name, :suffix,
+                     :youtube_account, :cspan_id, :votesmart_id, :icpsr_id,
+                     :crp_id, :google_entity_id, :fec_candidate_id, :rss_url,
+                     :ideal_point, :seniority, :next_election, :total_present,
+                     :ocd_id, :fax, :geoid]
+    rep_hash.delete_if {|key, value| unneeded_attr.include?(key)}
+  end
```
Comment (Rostammahabadi) [inline app/services/propublica_parser.rb:61]: This looks great!

**Review by Rostammahabadi — APPROVED** [review]

**Review by AntonioJacksonII — APPROVED** [review]
Looks good

File: app/services/propublica_parser.rb (hunk 1 above)
Comment (AntonioJacksonII) [inline app/services/propublica_parser.rb:52]: Great encapsulation!

File: db/schema.rb [hunk 3]
```diff
@@ -57,13 +57,13 @@
...
     t.integer "district"
-    t.string "at_large"
     t.float "missed_votes_percentage"
     t.float "votes_with_percentage"
-    t.string "votes_without_party_percentage"
     t.string "congress_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.float "votes_without_party_percentage"
+    t.boolean "at_large"
```
Comment (dborski) [inline db/schema.rb:66]: Looks good

File: spec/services/propublica_parser_spec.rb [hunk 4]
```diff
@@ -0,0 +1,75 @@
+require 'rails_helper'
```
Comment (dborski) [inline spec/services/propublica_parser_spec.rb:1]: Nice testing

**Review by dborski — APPROVED** [review]

**Pull request merged**